dev/routing/blocking.js

```diff
@@ -32,8 +34,14 @@ module.exports = function (server) {
         }
       },
       handler: function (request, reply) {
-        DAL.blocking.create(request.params.propertyId, request.payload, function (err, docs) {
-          !err ? reply(docs) : reply(JSON.stringify(err));
+        DAL.blocking.create(request.params.propertyId, request.payload, 
+        function (err, docs) {
+          DAL.properties.getById(request.params.propertyId, (err, property) => {
+            !err ? reply(towingBlockingService.sendNotification(
```

Comment:
"towingBlockingService.sendNotification" returns promise. Promose should not be parameter of "reply"